app/views/providers/means_reports/_caseworker_review.html.erb

```diff
@@ -1,34 +1,32 @@
 <section class="print-no-break">
   <h2 class="govuk-heading-l"><%= t("providers.means_reports.caseworker_review_section_heading") %></h2>
```

Comment:
I know this isn't changed but should it be?
All the other h2 classes have `<h2 class="govuk-heading-m">` but this one is `govuk-heading-l` 🤔

Reply:
so this matches similar pattern for dependants and  outgoings_and_deductions. I have checked with desihn and agreed to make all medium for consistency 🎉

Reply:
amended